protos/flyteidl/admin/launch_plan.proto

```diff
@@ -51,6 +52,9 @@ message LaunchPlan {
 
     // Values computed by the flyte platform after launch plan registration.
     LaunchPlanClosure closure = 3;
+
+    // DescriptionEntity contains detailed description for the launch plan.
+    DescriptionEntity description_entity = 4;
```

Comment:
I'm worried adding these inline is going to bloat the message size of these flyte entitity payloads by a lot
if necessary we can also add a List endpoint that supports the current semantics for filtering, sorting and pagination?

Reply:
sounds reasonable, removed it